akka-actor-typed/src/main/scala/akka/actor/typed/internal/ActorMdc.scala

```diff
@@ -13,13 +13,15 @@ import org.slf4j.MDC
 @InternalApi private[akka] object ActorMdc {
   val SourceKey = "akkaSource"
   val TagsKey = "akkaTags"
+  val SourceActorSystemKey = "sourceActorSystem"
 
   /**
    * @param tags empty string for no tags, a single tag or a comma separated list of tags
    */
-  def setMdc(source: String, tags: String): Unit = {
+  def setMdc(source: String, tags: String, actorSystemName: String): Unit = {
     val mdcAdapter = MDC.getMDCAdapter
     mdcAdapter.put(SourceKey, source)
+    mdcAdapter.put(SourceActorSystemKey, actorSystemName)
```

Comment:
so, the actor system name is not in the SourceKey because it's a local path?

What I would like is to include the address (host:port), since in any clustered system the systemName isn't adding much value as all are the same. Not sure if that is difficult to add here, but it would be more useful to have something like "ActorSysteName(host:port)" in a mdc entry

Reply:
Yeah, I'd like that as well, but didn't want to scope up this up to add that, the name is what is in the classic mdc for `sourceActorSystem`.

I'll create a separate issue to track that (and to add to both typed and classic)

Reply:
#28073